lib/wicked_pdf.rb

```diff
@@ -87,7 +87,7 @@ def pdf_from_url(url, options = {})
     generated_pdf_file.rewind
     generated_pdf_file.binmode
     pdf = generated_pdf_file.read
-    fail "PDF could not be generated!\n Command Error: #{err}" if pdf && pdf.rstrip.empty?
+    raise "PDF could not be generated!\n Command Error: #{err}" if pdf && pdf.rstrip.empty?
     pdf
```

Comment:
This was supposed to have been fixed in 2cfa57e331 but did not make it in for some reason.